controllers/glance_controller.go

```diff
@@ -246,6 +247,9 @@ func (r *GlanceReconciler) reconcileDelete(ctx context.Context, instance *glance
 		}
 	}
 
+	//TODO (fpantano): if quota limits are registered, we should remove.
```

Comment:
We should remove or update the registered limits?
Didn't understood this todo here.

Reply:
Done